package.json

```diff
@@ -20,19 +17,6 @@
     "prebuild": "npm run sass",
     "start": "npx @11ty/eleventy --serve --watch"
   },
-  "repository": {
```

Comment:
suggest keeping `repository`

Reply:
Given that this is an application, not a package, this field also seems unnecessary. What do you think?

Reply:
the repository field still works in applications with `npm repo`, so it's best to keep it.

Reply:
OK - I just restored it